
# Adjustment factor for safe and comfortable acceleration/deceleration
# Value type: Desired
# No unit
acceleration_adjustment_factor: 1.0
# Time horizon for collision detection for vehicles on the route
# NOTE: not applied for pedestrians or other non-vehicle obstacles on the road
# Units: s
# Value type: Desired
on_route_vehicle_collision_horizon_in_s: 10.0
# Minimum speed for moving obstacle
# Units: m/s
# Value type: Desired
obstacle_zero_speed_threshold_in_ms: 0.25
# Minimum object avoidance planning time
# Units: s
# Value type: Desired
min_obj_avoidance_plan_time_in_s: 2.0
# Deceleration value used for yielding trajectories
# Units: m/s^2
# Value type: Desired
yield_max_deceleration_in_ms2: 3.0
# Minimum safety gap with an Object/Obstacle
# Units: meters
# Value type: Desired
minimum_safety_gap_in_meters: 10.0
# Maximum speed value to consider the ego vehicle stopped
# Units: m/s
# Value type: Desired
max_stop_speed_in_ms: 1.0
# parameter to enable cooperative behavior
# Value type: Desired
enable_cooperative_behavior: false
# parameter to always accept mobility request
# Value type: Desired
always_accept_mobility_request: false
# Acceptable number of timesteps to use the latest known mobility request before switching to yield
# Value type: Desired
acceptable_passed_timesteps: 5
# Intervehicle distance that is considered a collision
# Unit: m
# Value type: Desired
intervehicle_collision_distance_in_m: 6.0
# Radius to check for potential collision
# Unit: m
# Value type: Desired
collision_check_radius_in_m: 150.0
# Time gap to finish planning a yield earlier than collision time
# Unit: s
# Value type: Desired
safety_collision_time_gap_in_s: 2.0
# Flag to enable yield plugin to check for adjustable gap (for example digital gap from map)
# Value type: Desired
enable_adjustable_gap: true
# Minimum urgency value to consider the mobility request
# Value type: Desired
acceptable_urgency: 5
# Window size for speed moving average filter
# Value type: Desired (must be odd)
speed_moving_average_window_size: 3.0
